tests/FSharp.Data.GraphQL.IntegrationTests.Server/FSharp.Data.GraphQL.IntegrationTests.Server.fsproj

```diff
@@ -7,17 +7,20 @@
 
   <ItemGroup>
     <PackageReference Include="Giraffe" />
+    <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Microsoft.Extensions.Http" />
+    <PackageReference Include="HotChocolate.AspNetCore" />
   </ItemGroup>
 
   <ItemGroup>
     <None Include="ApplicationInsights.config" />
-    <Compile Include="JsonConverters.fs" />
+    <Compile Include="JSON.fs" />
     <Compile Include="Helpers.fs" />
     <Compile Include="CustomSchemaTypes.fs" />
     <Compile Include="Schema.fs" />
-    <Compile Include="MultipartRequest.fs" />
```

Comment:
Don't we need it? It allows to perform file upload

Reply:
Added this back in on 2023/02/LHK-607-Introspection_Query .